### PR TITLE
Feature/cancel scheduled event frontend

### PR DESCRIPTION
**Pull request by aussie-m-mike**
https://trello.com/c/7mKWOfd3/366-cancel-event-button

### CODE REVIEW

File: client/src/store/modules/i18n/uk.js [hunk 1]
```diff
@@ -27,6 +27,8 @@ export default {
     CALENDAR_INVITATION_HAS_BEEN_SENT:
         'Запрошення до календаря надіслано на вашу електронну адресу',
     CANCEL: 'Скасувати',
+    CANCEL_EVENT_TEXT: 'Are you sure that you want cancel this event?',
+    CANCELLATION_EVENT: 'Cancellation event!',
```
Comment (yevhen-turovets) [inline client/src/store/modules/i18n/uk.js:31]: Why you don't use Ukraine language?

File: client/src/components/scheduled-events/Event.vue [hunk 2]
```diff
@@ -185,6 +218,14 @@ export default {
                 .padStart(2, '0');
 
             return day + ' ' + month + ' ' + year;
+        },
+
+        async onCancelHandle() {
+            const updatedEvent = {
+                ...this.scheduledEvent,
```
Comment (yevhen-turovets) [inline client/src/components/scheduled-events/Event.vue:225]: Why would you pass the entire state of the current event to change one field?

File: client/src/store/modules/scheduledEvent/normalizer.js [hunk 3]
```diff
@@ -22,3 +27,18 @@ export const eventPaginationMapper = Pagination => ({
...
+    invitee_email: Event.email,
+    invitee_question: Event.question,
+    status: Event.status,
+    start_date: moment(Event.startDate)
+        .tz(Event.timezone)
+        .utc()
+        .format('YYYY-MM-DD HH:mm:ss'),
+    location: Event.location,
+    timezone: Event.timezone,
+    event_type: eventTypeMapper(Event.eventType)
```
Comment (yevhen-turovets) [inline client/src/store/modules/scheduledEvent/normalizer.js:43]: I'm not sure if we have to process data on the client side

File: client/src/store/modules/scheduledEvent/actions.js [hunk 4]
```diff
@@ -113,5 +114,31 @@ export default {
...
         }
+    },
+
+    [actions.UPDATE_EVENT]: async ({ commit, dispatch }, event) => {
+        commit('loader/' + loaderMutations.SET_LOADING, true, { root: true });
+
+        try {
+            const updatedEvent = await scheduledEventService.updateEvent(
+                event.id,
+                eventApiMapper(event)
```
Comment (yevhen-turovets) [inline client/src/store/modules/scheduledEvent/actions.js:125]: I'm not sure if we have to process data on the client side

File: client/src/store/modules/scheduledEvent/actions.js [hunk 5]
```diff
@@ -113,5 +114,31 @@ export default {
                 root: true
             });
         }
+    },
+
+    [actions.UPDATE_EVENT]: async ({ commit, dispatch }, event) => {
```
Comment (yevhen-turovets) [inline client/src/store/modules/scheduledEvent/actions.js:119]: In this task, action should not be used to update all fields of the event - you need to update one field - the status.

File: client/src/store/modules/scheduledEvent/normalizer.js (hunk 3 above)
Comment (aussie-m-mike) [reply client/src/store/modules/scheduledEvent/normalizer.js:43]: We need push data from client in UTC format :)

File: client/src/components/scheduled-events/Event.vue (hunk 2 above)
Comment (aussie-m-mike) [reply client/src/components/scheduled-events/Event.vue:225]: I want use updateEvent action to the all update methods for event

File: client/src/store/modules/i18n/uk.js (hunk 1 above)
Comment (aussie-m-mike) [reply client/src/store/modules/i18n/uk.js:31]: I don't have Ukrainian keyboard layout in OS X

File: client/src/store/modules/scheduledEvent/actions.js (hunk 5 above)
Comment (aussie-m-mike) [reply client/src/store/modules/scheduledEvent/actions.js:119]: In this case we'll need to make a lot action update event, update start date, etc. In this way we could use just one and pass parameter to it.

File: client/src/store/modules/scheduledEvent/actions.js (hunk 4 above)
Comment (aussie-m-mike) [reply client/src/store/modules/scheduledEvent/actions.js:125]: We need to convert data, cause we have startDate on the frontend and we need to send start_date to backend

**Review by yevhen-turovets — APPROVED** [review]

**Review by yevhen-turovets — APPROVED** [review]

**Pull request merged**